src/js/hexbin/HexbinLayer.js

```diff
@@ -302,10 +302,30 @@ L.HexbinLayer = (L.Layer ? L.Layer : L.Class).extend({
 		this.options.onclick = clickFn;
 		//this._redraw();
 		return this;
+	},
+
+	/*
+	 * Returns an array of the points in the path, or nested arrays of points in case of multi-polyline.
+	 */
+	getLatLngs: function () {
+		return this._data.map(function(d) { return L.latLng(d[0], d[1]); });
```

Comment:
Ok thanks; happy for you to update during merge as you suggest.

Geometry *could* potentially be a multi string. 
For example if somebody were to use this for [MultiPolyline](http://leafletjs.com/reference-1.0.3.html#polyline), you would have a multidimensional array, that would then be handled by `L.GeoJSON.latLngsToCoords`